api/src/main/java/org/openmrs/module/pihcore/htmlformentry/action/CloseCovidProgramAction.java

```diff
@@ -0,0 +1,112 @@
+package org.openmrs.module.pihcore.htmlformentry.action;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.joda.time.DateTime;
+import org.openmrs.Encounter;
+import org.openmrs.Obs;
+import org.openmrs.Patient;
+import org.openmrs.PatientProgram;
+import org.openmrs.Program;
+import org.openmrs.api.ConceptService;
+import org.openmrs.api.ProgramWorkflowService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.emrapi.disposition.Disposition;
+import org.openmrs.module.emrapi.disposition.DispositionDescriptor;
+import org.openmrs.module.emrapi.disposition.DispositionService;
+import org.openmrs.module.htmlformentry.CustomFormSubmissionAction;
+import org.openmrs.module.htmlformentry.FormEntrySession;
+
+import java.util.Date;
+import java.util.List;
+
+public class CloseCovidProgramAction implements CustomFormSubmissionAction {
```

Comment:
`ExitFromCovidProgramAction`? Hopefully the hospital keeps the program open :)

Reply:
I can add "Patient", ie "ExitPatientFromCovidProgramAction"